src/components/NavMenu.tsx

```diff
@@ -7,15 +7,17 @@ import {
   typography,
   position,
   grid,
+  space,
   ColorProps,
   TypographyProps,
   PositionProps,
   GridProps,
+  SpaceProps,
 } from "styled-system";
 import { zIndexes } from "./theme";
 
 const overlayStyles = css`
-  display: flex;
+  display: -webkit-flex;
```

Comment:
styled components should do browser compatibility for you, was something not working?

Reply:
oooh my bad this was a change I made on another branch, I was trying to fix #145. fixed